pkg/apis/networking/types.go

```diff
@@ -195,6 +200,31 @@ type NetworkPolicyPeer struct {
 	IPBlock *IPBlock
 }
 
+// NetworkPolicyConditionType is the type for status conditions on
+// a NetworkPolicy. This type should be used with the
+// NetworkPolicyStatus.Conditions field.
+type NetworkPolicyConditionType string
+
+// NetworkPolicyConditionReason defines the set of reasons that explain why a
+// particular NetworkPolicy condition type has been raised.
+type NetworkPolicyConditionReason string
+
+const (
+	NetworkPolicyConditionStatusAccepted       NetworkPolicyConditionType = "Accepted"
+	NetworkPolicyConditionStatusPartialFailure NetworkPolicyConditionType = "PartialFailure"
```

Comment:
When would a CNI use PartialFailure? Does that mean it recognized some bad config but is still enforcing a policy based on it's best guess/capabilities?

Reply:
I've added some comments. Will leave this thread open so you folks can remember and review it :)